bitswap/src/protocol.rs

```diff
@@ -18,8 +18,8 @@ const MAX_BUF_SIZE: usize = 524_288;
 
 type FutureResult<T, E> = Pin<Box<dyn Future<Output = Result<T, E>> + Send>>;
 
-#[derive(Clone, Debug, Default)]
-pub struct BitswapConfig {}
+#[derive(Clone, Copy, Debug, Default)]
+pub struct BitswapConfig;
```

Comment:
Are we sure this will be `Copy` into the future? Wouldn't it be safer to leave this `!Copy`?

Reply:
it is cheaper for now; we are free to remove it later when this object is extended; adding anything non-`Copy` inside will immediately notify us, so there's no risk of forgetting it